### PR TITLE
Update zh_cn

### PR DESCRIPTION
**Pull request by Cactusstudent**

### CODE REVIEW

**Comment by Cactusstudent** [comment]
These changes can be merged. (thanks!)

File: src/main/resources/assets/tconstruct/book/materials_and_you/zh_cn/furtherreading/furtherreading.json [hunk 1]
```diff
@@ -1,6 +1,12 @@
 {
   "title": "延伸阅读",
   "text": [
-    { "text": "那些想要利用愚蠢出版交易规则，在我书前打广告的笨蛋们，你们失策了。我的书根本就不会出现在书架上，你自己就能制作出来。" }
+    {
+      "text": "现在，你已经了解了锻造的基础知识，花点时间来使用你学到的知识尝试制作一些基础工具。你一对这些基础知识感兴趣了之后，我的同事们就各自撰写了他们自己的几本书。同时，他们书中涵盖的话题范围更广。"
```
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/book/materials_and_you/zh_cn/furtherreading/furtherreading.json]:
```suggestion
      "text": "现在，你已经了解了一些基础知识，花点时间来使用你学到的知识尝试制作一些基础工具。你一对这些基础知识感兴趣了之后，我的同事们就各自撰写了他们自己的几本书。当然，他们书中涵盖的范围更广。"
```

File: src/main/resources/assets/tconstruct/book/puny_smelting/zh_cn/further_reading/mighty_smelting.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "title": "大师熔炼术",
+  "image": {
+    "file": "tconstruct:book/images/book_mighty_smelting.png",
+    "width": 128,
+    "height": 64
+  },
+  "text": [
+    {
+      "text": "有太多的精彩内容无法写进一本书，所以我不得不写第二本书。在你掌握了冶炼炉的使用之后，去熔炼一些砖泥，然后把它倒在一本书上，制作这本续集。"
```
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/book/puny_smelting/zh_cn/further_reading/mighty_smelting.json]:
```suggestion
      "text": "有太多的精彩内容无法写进一本书，所以我不得不将这些内容写进第二本书中。在你掌握了冶炼炉的使用之后，去熔炼一些砖泥，然后把它倒在一本书上，制作这本续集。"
```

**Comment by Cactusstudent** [comment]
btw, when i view the uk_ua.json (although i'm chinese), the 1134 miss a `,`, that will make the lang file dont work, maybe you can corret this()

**Comment by KnightMiner** [comment]
Please summarize these changes. It looks like you have quite a few changes to existing translations, can you briefly explain what changed to someone who does not speak that language? Is this mostly updates to the latest english version?

**Comment by Cactusstudent** [comment]
> Please summarize these changes. It looks like you have quite a few changes to existing translations, can you briefly explain what changed to someone who does not speak that language? Is this mostly updates to the latest english version?

1. Okay, these changes is mainly about the "挖掘" and “采掘” in Chinese, because in chinese, these two characters are similar, in TiC3, when we break any blocks, we usually use the "挖掘" (in enlish is mining). the "采掘" (in english is harvest) has a lot of meanings, and we use it to describe the "挖掘", "开采" and so on. so i discussed with the other translators about it, then we made a translate rule about the "mining" "harvest" and "mine", these changes are based on this. (here are the translate rules, but… it's chinese) (and these changes are too hard to describe in english because they only have a small difference in chinese, and i'm sorry about my bad writing enlish)

2. Some of these changes are update to the latest english version, some are just check and correct the grammar.

**Comment by Cactusstudent** [comment]
> > Please summarize these changes. It looks like you have quite a few changes to existing translations, can you briefly explain what changed to someone who does not speak that language? Is this mostly updates to the latest english version?
> 
> 1. Okay, these changes is mainly about the "挖掘" and “采掘” in Chinese, because in chinese, these two characters are similar, in TiC3, when we break any blocks, we usually use the "挖掘" (in enlish is mining). the "采掘" (in english is harvest) has a lot of meanings, and we use it to describe the "挖掘", "开采" and so on. so i discussed with the other translators about it, then we made a translate rule about the "mining" "harvest" and "mine", these changes are based on this. (here are the translate rules, but… it's chinese) (and these changes are too hard to describe in english because they only have a small difference in chinese, and i'm sorry about my bad writing enlish)
> 2. Some of these changes are update to the latest english version, some are just check and correct the grammar.

d990246b3f82ebe496d4bd2a6dce7969

File: src/main/resources/assets/tconstruct/book/materials_and_you/zh_cn/intro/partbuilder.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
   },
   "text": [
     {
-      "text": "制作部件极其简单：制作一个空白模具，将其放入模具槽。选择你想制作的工具部件，然后再放入一种徒手可以塑型的软性材料，就可以获得部件了。",
+      "text": "制作部件极其简单：制作一个空白模具，将其放入模具槽。选择你想制作的工具部件，然后再放入一种徒手可以塑型的软性材料，就可以获得部件了。You can also reshape a soft part you previously made.",
```
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/book/materials_and_you/zh_cn/intro/partbuilder.json]:
```suggestion
      "text": "制作部件极其简单：制作一个空白模具，将其放入模具槽。选择你想制作的工具部件，然后再放入一种徒手可以塑型的软性材料，就可以获得部件了。你同样可以将之前已经塑性的软性材料重新放入部件制造台制作新的工具部件。",
```

File: src/main/resources/assets/tconstruct/lang/zh_cn.json [hunk 4]
```diff
@@ -2429,7 +2428,11 @@
   "recipe.tconstruct.modifier.max_level": "%s的等级不可大于%s",
   "recipe.tconstruct.modifier.requirements_error": "工具的其中一个强化物已使该工具状态无效，联系整合包作者以了解详细错误情况。",
 
+  "recipe.tconstruct.tool_recycling": "Tool Recycling",
```
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/lang/zh_cn.json]:
```suggestion
  "recipe.tconstruct.tool_recycling": "工具回收",
```

File: src/main/resources/assets/tconstruct/lang/zh_cn.json [hunk 5]
```diff
@@ -2429,7 +2428,11 @@
...
+  "recipe.tconstruct.tool_recycling.info": "Select which part to salvage from this tool. There is a chance based on the tool's remaining durability to salvage a second part.",
```
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/lang/zh_cn.json]:
```suggestion
  "recipe.tconstruct.tool_recycling.info": "从构成此工具的工具部件中选择你想回收的一个。根据工具剩余的耐久值，有可能回收第二个工具部件。",
```

File: src/main/resources/assets/tconstruct/lang/zh_cn.json [hunk 6]
```diff
@@ -2429,7 +2428,11 @@
...
+  "recipe.tconstruct.tool_recycling.no_modifiers": "A tool cannot be recycled if it contains any upgrades or abilities, those must be removed first.",
```
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/lang/zh_cn.json]:
```suggestion
  "recipe.tconstruct.tool_recycling.no_modifiers": "施加强化或升级的工具无法被回收，你必须先将它们移除。",
```

File: src/main/resources/assets/tconstruct/book/materials_and_you/zh_cn/intro/recycling.json [hunk 7]
```diff
@@ -0,0 +1,14 @@
+{
+  "title": "Recycling Tools",
```
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/book/materials_and_you/zh_cn/intro/recycling.json]:
```suggestion
  "title": "回收工具",
```

File: src/main/resources/assets/tconstruct/book/materials_and_you/zh_cn/intro/recycling.json [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+  "image": {
+    "file": "tconstruct:book/images/recycling.png",
+    "width": 170,
+    "height": 64
+  },
+  "text": [
+    {
+      "text": "If you change your mind about a tool, you can always disassemble it in the part builder to salvage one of the tool parts. You can potentially salvage a second part, with a higher chance on repaired tools.",
```
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/book/materials_and_you/zh_cn/intro/recycling.json]:
```suggestion
      "text": "如果你突然对制作工具有了别的想法，你可以将工具放入部件制造台拆解以回收其中一个工具部件。同样，你也有可能获得第二个部件，当你修复工具后，获得第二个部件的几率会大大增加。",
```

**Review by Cactusstudent — COMMENTED** [review]
Translate the zh_cn and book

**Review by Cactusstudent — COMMENTED** [review]
写反了……应该是耐久值->耐久度

File: src/main/resources/assets/tconstruct/book/materials_and_you/zh_cn/furtherreading/fantastic_foundry_craft.json [hunk 9]
```diff
@@ -1,5 +1,5 @@
 {
-  "title": "奇异熔铸术",
+  "title": "《奇异熔铸术》 Nemea著",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/book/materials_and_you/zh_cn/furtherreading/fantastic_foundry_craft.json]:
```suggestion
  "title": "《奇异熔铸术》Nemea著",
```

File: src/main/resources/assets/tconstruct/book/materials_and_you/zh_cn/furtherreading/puny_craft.json [hunk 10]
```diff
@@ -1,8 +1,8 @@
 {
-  "title": "入门熔炼术",
+  "title": "《入门熔炼术》 Thruul著",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/book/materials_and_you/zh_cn/furtherreading/puny_craft.json]:
```suggestion
  "title": "《入门熔炼术》Thruul著",
```

File: src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/abilities/armor/pockets.json [hunk 11]
```diff
@@ -7,7 +7,7 @@
   ],
   "effects": [
     "增加更多物品栏槽位",
-    "潜行状态下按“护腿交互”键打开护腿所属物品栏",
+    "同时按下Shift键与“护腿交互”键打开护腿所属物品栏",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/abilities/armor/pockets.json]:
```suggestion
    "同时按下左Shift键与“护腿交互”键打开护腿所属物品栏",
```
Depends on whether the Left Shift is specified.

File: src/main/resources/assets/tconstruct/lang/zh_cn.json [hunk 12]
```diff
@@ -1332,9 +1331,9 @@
   "material.tconstruct.wood.jungle": "丛林木",
   "material.tconstruct.wood.acacia": "金合欢木",
   "material.tconstruct.wood.dark_oak": "深色橡木",
-  "material.tconstruct.wood.crimson": "绯红菌核",
-  "material.tconstruct.wood.warped": "诡异菌核",
-  "material.tconstruct.wood.bamboo": "竹子",
+  "material.tconstruct.wood.crimson": "绯红菌木",
+  "material.tconstruct.wood.warped": "诡异菌木",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/lang/zh_cn.json]:
```suggestion
  "material.tconstruct.wood.crimson": "绯红木",
  "material.tconstruct.wood.warped": "诡异木",
```

File: src/main/resources/assets/tconstruct/lang/zh_cn.json [hunk 13]
```diff
@@ -1863,13 +1862,14 @@
   "modifier.tconstruct.leaping.description": "兔子可以跳得很高，那么这个应该也能让你跳得很高",
   "modifier.tconstruct.pockets": "行囊",
   "modifier.tconstruct.pockets.flavor": "真是奇妙的发明！",
-  "modifier.tconstruct.pockets.description": "这护腿是有口袋的，能让你存储一些额外的物品，潜行时按下“护腿交互”键就可以存取",
+  "modifier.tconstruct.pockets.description": "这护腿是有口袋的，能让你存储一些额外的物品，潜行时按下“护腿交互”键就可以存取 The pants have pockets, allowing you to store additional items which can be accessed by shift plus interacting",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/lang/zh_cn.json]:
```suggestion
  "modifier.tconstruct.pockets.description": "这护腿是有口袋的，能让你存储一些额外的物品，潜行时按下“护腿交互”键就可以存取",
```

File: src/main/resources/assets/tconstruct/lang/zh_cn.json [hunk 14]
```diff
@@ -2046,22 +2046,22 @@
 
   "modifier.tconstruct.cultivated": "培植",
   "modifier.tconstruct.cultivated.flavor": "精打细算！",
-  "modifier.tconstruct.cultivated.description": "工具在修复时，单位材料实际能够提供更多耐久值。",
+  "modifier.tconstruct.cultivated.description": "工具在修复时，单位材料实际能够提供更多耐久度。",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/lang/zh_cn.json]:
```suggestion
  "modifier.tconstruct.cultivated.description": "工具在修复时，单位材料能够提供更多耐久度。",
```

File: src/main/resources/assets/tconstruct/lang/zh_cn.json [hunk 15]
```diff
@@ -1863,13 +1862,14 @@
...
-  "modifier.tconstruct.pockets.description": "这护腿是有口袋的，能让你存储一些额外的物品，潜行时按下“护腿交互”键就可以存取",
+  "modifier.tconstruct.pockets.description": "这护腿是有口袋的，能让你存储一些额外的物品，潜行时按下“护腿交互”键就可以存取 The pants have pockets, allowing you to store additional items which can be accessed by shift plus interacting",
   "modifier.tconstruct.shield_strap": "副手套环",
   "modifier.tconstruct.shield_strap.flavor": "保护与攻击的理想之选",
   "modifier.tconstruct.shield_strap.description": "按下“护腿交互”键能够交换副手物品",
   "modifier.tconstruct.tool_belt": "工具腰带",
   "modifier.tconstruct.tool_belt.flavor": "恨不得长出六只手么？",
-  "modifier.tconstruct.tool_belt.description": "按下护腿交互键能够交换快捷栏",
+  "modifier.tconstruct.tool_belt.description": "按下护腿交互键能够交换快捷栏 Interacting with the leggings whle holding control allows swapping the hotbar",
+  "recipe.tconstruct.modifier.tool_belt": "Tool belt recipes must be applied in order",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/lang/zh_cn.json]:
```suggestion
  "modifier.tconstruct.tool_belt.description": "按下护腿交互键能够交换快捷栏",
  "recipe.tconstruct.modifier.tool_belt": "Tool belt recipes must be applied in order",
```

```suggestion
  "modifier.tconstruct.tool_belt.description": "按下护腿交互键能够交换快捷栏",
  "recipe.tconstruct.modifier.tool_belt": "“工具腰带”需按顺序施加",
```
*Can't find where the latter key would appear*

Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/lang/zh_cn.json]:
```suggestion
  "recipe.tconstruct.modifier.tool_belt": "“工具腰带”需按顺序施加",
```

File: src/main/resources/assets/tconstruct/book/fantastic_foundry/zh_cn/foundry/controller.json [hunk 16]
```diff
@@ -5,7 +5,7 @@
   },
   "text": [
     {
-      "text": "将1块量的熔融黑曜石浇注到焦褐砖块上，你就能获得这个神秘的熔铸炉控制器。"
+      "text": "将1块量的熔融黑曜石浇注到焦褐石砖上，你就能获得这个神秘的熔铸炉控制器。"
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/book/fantastic_foundry/zh_cn/foundry/controller.json]:
```suggestion
      "text": "将1块量的熔融黑曜石浇注到焦褐砖块上，你就能获得这个神秘的熔铸炉控制器。"
```

File: src/main/resources/assets/tconstruct/book/puny_smelting/zh_cn/slotless/general/overslime.json [hunk 17]
```diff
@@ -6,7 +6,7 @@
     }
   ],
   "effects": [
-    "增加一层黏液覆层，其优先于耐久被消耗",
+    "增加1层黏液覆层，其优先于耐久被消耗",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/book/puny_smelting/zh_cn/slotless/general/overslime.json]:
```suggestion
    "增加1层黏液覆层，其优先于耐久度被消耗",
```

File: src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/tools/small/pickadze.json [hunk 18]
```diff
@@ -1,16 +1,16 @@
 {
   "tool": "tconstruct:pickadze",
   "text": [
-    { "text": "凿石锹是一种多用途的石头和泥土挖掘工具，对多种方块都有效，但挖掘等级无法应用到这些方块上。" },
+    { "text": "凿石锹是一种多用途的石头和泥土挖掘工具，对多种方块都有效，但采掘等级无法应用到这些方块上。" },
```
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/tools/small/pickadze.json]:
```suggestion
    { "text": "凿石锹是一种多用途的石头和泥土挖掘工具，对多种方块都有效，但采掘等级无法应用到此工具上。" },
```

File: src/main/resources/assets/tconstruct/book/materials_and_you/zh_cn/intro/partbuilder.json [hunk 19]
```diff
@@ -7,7 +7,7 @@
   },
   "text": [
     {
-      "text": "制作部件极其简单：制作一个空白模具，将其放入模具槽。选择你想制作的工具部件，然后再放入一种徒手可以塑型的软性材料，就可以获得部件了。",
+      "text": "制作部件极其简单：制作一个空白模具，将其放入模具槽。选择你想制作的工具部件，然后再放入一种徒手可以塑型的软性材料，就可以获得部件了。你同样可以将之前已经塑性的软性材料重新放入部件制造台制作新的工具部件。",
```
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/book/materials_and_you/zh_cn/intro/partbuilder.json]:
```suggestion
      "text": "制作部件极其简单：制作一个空白模具，将其放入模具槽。选择你想制作的工具部件，然后再放入一种徒手可以塑型的软性材料，就可以获得部件了。你同样可以将之前已经塑形的软性材料重新放入部件制造台制作新的工具部件。",
```

File: src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/abilities/general/unbreakable.json [hunk 20]
```diff
@@ -8,7 +8,7 @@
   "effects": [
     "你的工具现在牢不可破",
     "单个等级",
-    "需要1个能力槽、“加固 V”和“下界合金”",
-    "不免疫啃食造成的耐久值降低"
+    "需要1个能力槽、以及“加固 V”和“下界合金”作为前置",
+    "不免疫啃食造成的耐久度值降低"
```
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/abilities/general/unbreakable.json]:
```suggestion
    "不免疫啃食造成的耐久度降低"
```

**Comment by ManyullynSword** [comment]
I think the "necrotic" modifier in the "Encyclopedia" Chinese the "暴击" in the introduction should be changed to "重击"，I think it's more respectful of vanilla.

**Comment by Cactusstudent** [comment]
But… in vanilla Minecraft, we use the "暴击”…

**Comment by ManyullynSword** [comment]
"重击" is the official language of vanilla.

**Comment by ChuijkYahus** [comment]
> "重击" is the official language of vanilla.


If it's `Critical Hit` you're saying, it seems `暴击` is the word officially used in Vanilla versions.
图片
图片
图片
图片

**Comment by Cactusstudent** [comment]
FINISH!!!!

**Comment by shshenjiang** [comment]
...... The name of the "fareward.json" file in the "encyclopedia" intro section is written as "farward.json".

**Comment by ManyullynSword** [comment]
...... The name of the "foreward.json" file in the "encyclopedia" intro section is written as "forward.json",So much so that this section shows English.

**Comment by Cactusstudent** [comment]
> ...... The name of the "foreward.json" file in the "encyclopedia" intro section is written as "forward.json",So much so that this section shows English.

We have finished it maybe a month ago
大概……一个多月前修了吧……只是由于PR还没合并……所以就还是英文

**Review by Cactusstudent — COMMENTED** [review]
Review-1

File: src/main/resources/assets/tconstruct/lang/zh_cn.json [hunk 21]
```diff
@@ -2429,23 +2429,23 @@
...
+  "recipe.tconstruct.modifier_sorting.not_enough_modifiers": "工具至少需要2种强化以排序",
 
-  "recipe.tconstruct.invisible_modifiers.adding.title": "Invisible Ink",
-  "recipe.tconstruct.invisible_modifiers.adding.description": "Select a modifier to hide it from the tool's model",
-  "recipe.tconstruct.invisible_modifiers.removing.title": "Visible Ink",
-  "recipe.tconstruct.invisible_modifiers.removing.description": "Select a hidden modifier to make it show in the tool's model agin",
+  "recipe.tconstruct.invisible_modifiers.adding.title": "隐形墨水",
+  "recipe.tconstruct.invisible_modifiers.adding.description": "选择一个强化以将其材质从工具模型中隐藏",
+  "recipe.tconstruct.invisible_modifiers.removing.title": "显性墨水",
+  "recipe.tconstruct.invisible_modifiers.removing.description": "选择一个隐藏的强化以使其从工具模型中再次显性",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/lang/zh_cn.json]:
```suggestion
  "recipe.tconstruct.invisible_modifiers.removing.title": "显形墨水",
  "recipe.tconstruct.invisible_modifiers.removing.description": "选择一个隐藏的强化以使其从工具模型中再次显形",
```
“形” as in "figure, shape, form"

Comment (Cactusstudent) [reply src/main/resources/assets/tconstruct/lang/zh_cn.json]: thx

File: src/main/resources/assets/tconstruct/lang/zh_cn.json [hunk 22]
```diff
@@ -1989,6 +1989,18 @@
...
+  "modifier.tconstruct.shears.description": "Tool can be used to shear mobs and disarm tripwire",
+  "modifier.tconstruct.silky_shears": "Silky Shears",
+  "modifier.tconstruct.silky_shears.flavor": "Snip snip!",
+  "modifier.tconstruct.silky_shears.description": "Tool can shear mobs if the silky modifier is added",
+  "modifier.tconstruct.silky_aoe_shears": "Silky Shears",
+  "modifier.tconstruct.silky_aoe_shears.flavor": "Snip snip!",
+  "modifier.tconstruct.silky_aoe_shears.description": "Tool can shear mobs if the silky modifier is added",
+  "modifier.tconstruct.harvest": "Harvest",
+  "modifier.tconstruct.harvest.flavor": "Swish!",
+  "modifier.tconstruct.harvest.description": "Tool can be used on crops to harvest and replant them",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/lang/zh_cn.json]:
```suggestion
  "modifier.tconstruct.shears": "修剪",
  "modifier.tconstruct.shears.flavor": "咔嚓咔嚓！",
  "modifier.tconstruct.shears.description": "工具可用于修剪生物和剪除绊线",
  "modifier.tconstruct.silky_shears": "丝剪",
  "modifier.tconstruct.silky_shears.flavor": "咔嚓咔嚓！",
  "modifier.tconstruct.silky_shears.description": "若加持有“丝触”强化，则工具可用于修剪生物",
  "modifier.tconstruct.silky_aoe_shears": "丝剪",
  "modifier.tconstruct.silky_aoe_shears.flavor": "咔嚓咔嚓！",
  "modifier.tconstruct.silky_aoe_shears.description": "若加持有“丝触”强化，则工具可用于修剪生物",
  "modifier.tconstruct.harvest": "收割",
  "modifier.tconstruct.harvest.flavor": "春耕秋获！",
  "modifier.tconstruct.harvest.description": "工具可收割并重新种植农作物",
```
Translating `Swish!` with a single interjection seems quite off to me.
This version of translation of that key means literally "sow/plough in spring, harvest in autumn"

Comment (Cactusstudent) [reply src/main/resources/assets/tconstruct/lang/zh_cn.json]: pretty good, maybe we should discuss the `Swish!`
看起来好极了！我们可能得讨论下`Swish!` 的中文翻译

**Review by ChuijkYahus — APPROVED** [review]

**Comment by ManyullynSword** [comment]
There is a text error that many people have, and there is no tick between the book titles!
有一个很多人都有的文本错误，书名号之间是没有顿号的！

**Comment by Cactusstudent** [comment]
> There is a text error that many people have, and there is no tick between the book titles! 有一个很多人都有的文本错误，书名号之间是没有顿号的！

Okey, i'll fix this

**Comment by Cactusstudent** [comment]
> > There is a text error that many people have, and there is no tick between the book titles! 有一个很多人都有的文本错误，书名号之间是没有顿号的！
> 
> Okey, i'll fix this

但这玩意在哪里啊？能否分别指明一下，不胜感激

**Comment by ManyullynSword** [comment]
那直接说中文咯，“工匠大百科”中的“内容更新”。

**Comment by Cactusstudent** [comment]
> 那直接说中文咯，“工匠大百科”中的“更进一步”。

成，那我现在就一修

**Comment by ManyullynSword** [comment]
啊，不对。是“内容更新”！

**Comment by ManyullynSword** [comment]
The "Small Blade" should not be called the "小型剑刃", because it can also be used to synthesize "Dagger" and "Kama".
那个“Small Blade”不应该叫“小型剑刃”，因为其还可以用于合成“匕首”和“短刃镰”。

**Comment by HaloSense** [comment]
Are contents of this PR in sync of that in my repo? I am planning on updating the contents of my repo recently, in accordance with updated contents of the mod, and I need to make sure everything in my repo is up to date. 
这个PR里的翻译内容和我库里的内容同步吗？我最近打算根据模组更新的内容对我库里的翻译进行一次大修补，修补之前我得保证库里的内容是最新的

**Comment by Cactusstudent** [comment]
> Are contents of this PR in sync of that in my repo? I am planning on updating the contents of my repo recently, in accordance with updated contents of the mod, and I need to make sure everything in my repo is up to date. 这个PR里的翻译内容和我库里的内容同步吗？我最近打算根据模组更新的内容对我库里的翻译进行一次大修补，修补之前我得保证库里的内容是最新的

这里的可以随便用，是追着GitHub的Commit做的更新（即最新的）
和你的仓库可能偏差也很大了，因为在这段时间更改了很多用词，也新增/删除了很多翻译

**Comment by HaloSense** [comment]
> > Are contents of this PR in sync of that in my repo? I am planning on updating the contents of my repo recently, in accordance with updated contents of the mod, and I need to make sure everything in my repo is up to date. 这个PR里的翻译内容和我库里的内容同步吗？我最近打算根据模组更新的内容对我库里的翻译进行一次大修补，修补之前我得保证库里的内容是最新的
> 
> 这里的可以随便用，是追着GitHub的Commit做的更新（即最新的） 和你的仓库可能偏差也很大了，因为在这段时间更改了很多用词，也新增/删除了很多翻译

脑袋疼……这得更新到啥时候去

**Comment by Cactusstudent** [comment]
> > > Are contents of this PR in sync of that in my repo? I am planning on updating the contents of my repo recently, in accordance with updated contents of the mod, and I need to make sure everything in my repo is up to date. 这个PR里的翻译内容和我库里的内容同步吗？我最近打算根据模组更新的内容对我库里的翻译进行一次大修补，修补之前我得保证库里的内容是最新的
> > 
> > 
> > 这里的可以随便用，是追着GitHub的Commit做的更新（即最新的） 和你的仓库可能偏差也很大了，因为在这段时间更改了很多用词，也新增/删除了很多翻译
> 
> 脑袋疼……这得更新到啥时候去

话说要本地的文件吗？我可以邮箱发你（虽然Clone到本地也行）

**Comment by HaloSense** [comment]
> > > > Are contents of this PR in sync of that in my repo? I am planning on updating the contents of my repo recently, in accordance with updated contents of the mod, and I need to make sure everything in my repo is up to date. 这个PR里的翻译内容和我库里的内容同步吗？我最近打算根据模组更新的内容对我库里的翻译进行一次大修补，修补之前我得保证库里的内容是最新的
> > > 
> > > 
> > > 这里的可以随便用，是追着GitHub的Commit做的更新（即最新的） 和你的仓库可能偏差也很大了，因为在这段时间更改了很多用词，也新增/删除了很多翻译
> > 
> > 
> > 脑袋疼……这得更新到啥时候去
> 
> 话说要本地的文件吗？我可以邮箱发你

不用……你想个办法更新一下我库里那边的文件就行了，之后大概一个月来这么一次吧，如果你打算继续维护这个pr的话

**Comment by Cactusstudent** [comment]
> 不用……你想个办法更新一下我库里那边的文件就行了，之后大概一个月来这么一次吧，如果你打算继续维护这个pr的话

行，这几天整理下PR过去

**Comment by HaloSense** [comment]
我是觉得一直这么追着官方库翻译有点太累了，保证咱们的内容在一定程度内是最新就行了

**Comment by KnightMiner** [comment]
Is tis in a state where I can merge it for the soon to come build, or would you rather I hold off until the inevitable bug fix release for bows?

**Comment by Cactusstudent** [comment]
Sorry to be late. We haven't finished yet and we'll try to finish it in a day. If not, you can merge it (because i have finished updating) 
Most of the documents have been translated.

**Comment by KnightMiner** [comment]
The author of #4753 said that Chinese may find the ability to set the separator between tool material and name useful (its no longer hardcoded to `<material> <tool name>`. They wanted it for Japanese. If you want more info on how you can use that feature, check the most recent comment from me.

**Comment by Cactusstudent** [comment]
~~Maybe we finish?~~ We Finished it!

**Pull request merged**